exporter/kafkaexporter/config_modified_test.go

```diff
@@ -81,5 +81,11 @@ func TestLoadConfig(t *testing.T) {
 		},
 		Debug:              true,
 		DumpSpanAttributes: true,
+		SpanCuring: SpanCuring{
+			Enabled:               true,
+			DropSpans:             true,
+			MaxAttributeValueSize: 131072,
```

Comment:
Is this in bytes, right?

Reply:
yes in bytes

Reply:
will rename to `max_attribute_value_size bytes` in a chore PR

Reply:
https://github.com/hypertrace/hypertrace-collector/pull/72